bin/desi_fit_stdstars.py

```diff
@@ -79,7 +79,6 @@ def main() :
     filters=fibers["FILTER"]
     if 'DESISIM' not in os.environ:
         raise RuntimeError('Set environment DESISIM. Can not find filter response files')
```

Comment:
Does DESISIM still need to be defined?  If so, this exception message should be updated.